client/devdocs/design/typography.jsx

```diff
@@ -48,21 +48,21 @@ export default React.createClass( {
 		};
 
 		const contentTitle = {
-			fontFamily: 'Merriweather',
+			fontFamily: 'Noto Serif',
```

Comment:
@mtias I know the `$serif` variable is available in Sass, but not sure if we can use it in JSX components?